backend/WikiContrib/result/views.py

```diff
@@ -74,11 +74,12 @@ def fuzzyMatching(control, full_names):
     _list = []
     ave = 0
     for key in full_names:
-        if (full_names[key] != username_does_not_exist and
-        full_names[key] != no_username_provided):
-            _list.append(fuzz.WRatio(control, full_names[key]))
+        if full_names[key] != username_does_not_exist:
+            if full_names[key] != no_username_provided:
+                _list.append(fuzz.WRatio(control, full_names[key],score_cutoff=60))
```

Comment:
Can you explain what `score_cutoff=60` means here? And, we are setting it as 60?

Reply:
We take to be an acceptable match percentage is 60%. Anything below that and the warning will be triggered. score_cutoff here is an improvement over the way the fuzzy matching used to work in the past. It ensures that we still get the same behavior when comparing one username, as well as two usernames. for example:

fullname = reedy
phab_username = dbrant
gerrit_username = reedy

(dbrant full name is Dmitry Brant , reedy's fullname is reedy)

# without cut_off argument

phab_match_percentage = 36 (well below the acceptable %. will trigger warning if we only provided phab_username)
gerrit_match_percentage = 100 (well above the acceptable %. Will not trigger warning if we only provided gerrit_username)

but then, since we provided both usernames, we need to find the average.
average = (36+100)/2 => 68%
one of the usernames is wrong, but since we didn't specify cut-off, no warning will be thrown


# with cut_off argument

phab_match_percentage = 0 (cut_off acts like binary, below which the percentage is rounded down to zero)
gerrit_match_percentage = 100 (cut_off acts like binary, above which the percentage is rounded up to 100)

average = (0+100)/2 => 50%
since average is 50%, warning will be triggered. This is the expected behavior


It might be worth mentioning that the cut_off argument is an improvement over the whole fuzzy matching feature and not just limited to the issue this PR is addressing. I we run the above test before merging this PR we will still experience the problem pointed out above